src/devices/QwiicButton/QwiicButton.InterruptConfiguration.cs

```diff
@@ -0,0 +1,92 @@
+﻿//// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
use double `/` instead of quadra `/` (here and everywhere)

Reply:
Fixed